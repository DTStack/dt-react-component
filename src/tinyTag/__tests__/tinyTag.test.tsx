import React from 'react';
import { cleanup, render } from '@testing-library/react';

import TinyTag from '..';

describe('test StatusTag', () => {
    beforeEach(cleanup);
    test('should match snapshot', () => {
        const { asFragment } = render(<TinyTag value="完成" className="dtc-test" />);
        expect(asFragment()).toMatchSnapshot();
    });
});
