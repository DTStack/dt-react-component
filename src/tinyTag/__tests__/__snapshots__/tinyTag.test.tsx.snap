// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test StatusTag should match snapshot 1`] = `
<DocumentFragment>
  <span
    class="dtc-tinyTag dtc-test"
  >
    完成
  </span>
</DocumentFragment>
`;
